src/lib.rs

```diff
@@ -573,21 +573,24 @@ impl Certificate {
 			});
 			// Write subjectPublicKeyInfo
 			self.key_pair.serialize_public_key_der(writer.next());
-			// Write extensions
-			if !self.params.subject_alt_names.is_empty() {
-				writer.next().write_tagged(Tag::context(0), |writer| {
-					writer.write_sequence(|writer| {
-						let oid = ObjectIdentifier::from_slice(OID_PKCS_9_AT_EXTENSION_REQUEST);
-						writer.next().write_oid(&oid);
-						writer.next().write_set(|writer| {
-							writer.next().write_sequence(|writer| {
-								// Write subject_alt_names
-								self.write_subject_alt_names(writer.next());
-							});
-						});
-					});
-				});
-			}
+            // Write extensions
+            // To match what openssl outputs and what some asn1 decoders expect, always write the
```

Comment:
In this instance it's even a spec violation, so could you remove the openssl and decoder references from this comment and instead say that it's expected by the spec?

Reply:
All fixed!

Reply:
Are you sure? I'm still seeing the old version.

Reply:
Ha, oops. This is what happens when you forget to `git add`. Now we're good